### PR TITLE
Update airbase to 156

### PR DESCRIPTION
**Pull request by wendigo**
~OpenTelemetry stuff is temporary.~

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -214,6 +214,46 @@
 
     <dependencyManagement>
         <dependencies>
+            <?SORTPOM IGNORE?>
+            <!--
+                Temporary workaround to settle opentelemetry on 1.37.0 version.
```
Comment (martint) [inline pom.xml:219]: How temporary? 

BTW, if the order the declarations appear in the dependencyManagement section matters (ouch!), then it's improper to try to sort them. Should we get rid of sorting altogether?

Comment (wendigo) [reply pom.xml:219]: until the `libraries-bom` are updated (the chain of BOM imports is: libraries-bom -> first-party-dependencies bom -> gapic-generator-java bom -> opentelemetry bom)

Comment (wendigo) [reply pom.xml:219]: The order of the declarations matter for BOMs which imports their dependencies into the current scope and first import wins.

Comment (wendigo) [reply pom.xml:219]: I think that the new libraries-bom is coming soon and it should contain new opentelemetry import that doesn't downgrade our own requirement.

Comment (wendigo) [reply pom.xml:219]: https://github.com/googleapis/java-cloud-bom/releases/tag/v26.38.0 was tagged an hour ago

Comment (wendigo) [reply pom.xml]: Just in time. Dropped.

**Comment by wendigo** [comment]
@martint PTAL

**Review by martint — APPROVED** [review]

**Pull request merged**